lib/util/middlewares.js

```diff
@@ -74,7 +74,7 @@ exports.swigFilters = function(app, swig) {
       }
 
       // ページの末尾を拾う
-      const suffix = name.replace(/.+\/(.+)?$/, '$1')
+      const suffix = name.replace(/.+\/(.+)$/, '$1')
```

Comment:
`path2name` filter is equivalent to `getShortPath` method. So this method should be changed.

In the future, I believe that we can delete `path2name` filter by replacing Swig templates with React.
For that reason, I've copied and pasted as a temporary measure.

I can use Suggest for changes, but `middlewares.js` was not changed in this pull request, so I committed directly.
https://help.github.com/en/articles/reviewing-proposed-changes-in-a-pull-request

Reply:
Oh, thanks!